### PR TITLE
feat(plasma-tokens,plasma-tokens-utils): Add generating brand for each themes

### PR DESCRIPTION
**Pull request by neretin-trike**
:baby_chick: Download canary assets:

[colors--canary.343.4182782211.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.343.4182782211.xml)  
[color_metro_ios-swift--canary.343.4182782211.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.343.4182782211.swift)  
[color_metro_kotlin--canary.343.4182782211.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.343.4182782211.kt)  
[color_metro_react-native--canary.343.4182782211.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.343.4182782211.ts)  
[color_sbermarket_ios-swift--canary.343.4182782211.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.343.4182782211.swift)  
[color_sbermarket_kotlin--canary.343.4182782211.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.343.4182782211.kt)  
[color_sbermarket_react-native--canary.343.4182782211.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.343.4182782211.ts)  
[color_sberprime_ios-swift--canary.343.4182782211.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.343.4182782211.swift)  
[color_sberprime_kotlin--canary.343.4182782211.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.343.4182782211.kt)  
[color_sberprime_react-native--canary.343.4182782211.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.343.4182782211.ts)  
[color_selgros_ios-swift--canary.343.4182782211.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.343.4182782211.swift)  
[color_selgros_kotlin--canary.343.4182782211.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.343.4182782211.kt)  
[color_selgros_react-native--canary.343.4182782211.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.343.4182782211.ts)  
[color_smbusiness_ios-swift--canary.343.4182782211.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.343.4182782211.swift)  
[color_smbusiness_kotlin--canary.343.4182782211.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.343.4182782211.kt)  
[color_smbusiness_react-native--canary.343.4182782211.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.343.4182782211.ts)  
[PlasmaTokensColor--canary.343.4182782211.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.343.4182782211.swift)  
[shadow_sbermarket_react-native--canary.343.4182782211.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.343.4182782211.ts)  
[typo_mage_ios-swift--canary.343.4182782211.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.343.4182782211.swift)  
[typo_mage_kotlin--canary.343.4182782211.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.343.4182782211.kt)  
[typo_mage_react-native--canary.343.4182782211.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.343.4182782211.ts)  
[typo_plasma_ios-swift--canary.343.4182782211.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.343.4182782211.swift)  
[typo_plasma_kotlin--canary.343.4182782211.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.343.4182782211.kt)  
[typo_plasma_react-native--canary.343.4182782211.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.343.4182782211.ts)  
[typo_ruler_ios-swift--canary.343.4182782211.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.343.4182782211.swift)  
[typo_ruler_kotlin--canary.343.4182782211.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.343.4182782211.kt)  
[typo_ruler_react-native--canary.343.4182782211.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.343.4182782211.ts)  
[typo_sage_ios-swift--canary.343.4182782211.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.343.4182782211.swift)  
[typo_sage_kotlin--canary.343.4182782211.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.343.4182782211.kt)  
[typo_sage_react-native--canary.343.4182782211.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.343.4182782211.ts)  
[typo_sbermarket_ios-swift--canary.343.4182782211.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.343.4182782211.swift)  
[typo_sbermarket_kotlin--canary.343.4182782211.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.343.4182782211.kt)  
[typo_sbermarket_react-native--canary.343.4182782211.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.343.4182782211.ts)  
[typo_soulmate_ios-swift--canary.343.4182782211.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.343.4182782211.swift)  
[typo_soulmate_kotlin--canary.343.4182782211.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.343.4182782211.kt)  
[typo_soulmate_react-native--canary.343.4182782211.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.343.4182782211.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.140.2-canary.343.4182782211.0
  npm install @salutejs/plasma-core@1.96.1-canary.343.4182782211.0
  npm install @salutejs/plasma-icons@1.124.1-canary.343.4182782211.0
  npm install @salutejs/plasma-temple@1.135.2-canary.343.4182782211.0
  npm install @salutejs/plasma-tokens-b2b@1.9.1-canary.343.4182782211.0
  npm install @salutejs/plasma-tokens-b2c@0.18.1-canary.343.4182782211.0
  npm install @salutejs/plasma-tokens-web@1.24.1-canary.343.4182782211.0
  npm install @salutejs/plasma-tokens@1.35.1-canary.343.4182782211.0
  npm install @salutejs/plasma-ui@1.168.2-canary.343.4182782211.0
  npm install @salutejs/plasma-web@1.166.2-canary.343.4182782211.0
  npm install @salutejs/plasma-cy-utils@0.42.1-canary.343.4182782211.0
  npm install @salutejs/plasma-sb-utils@0.94.1-canary.343.4182782211.0
  npm install @salutejs/plasma-tokens-android@2.42.1-canary.343.4182782211.0
  npm install @salutejs/plasma-tokens-ios-swift@2.42.1-canary.343.4182782211.0
  npm install @salutejs/plasma-tokens-utils@0.17.1-canary.343.4182782211.0
  # or 
  yarn add @salutejs/plasma-b2c@1.140.2-canary.343.4182782211.0
  yarn add @salutejs/plasma-core@1.96.1-canary.343.4182782211.0
  yarn add @salutejs/plasma-icons@1.124.1-canary.343.4182782211.0
  yarn add @salutejs/plasma-temple@1.135.2-canary.343.4182782211.0
  yarn add @salutejs/plasma-tokens-b2b@1.9.1-canary.343.4182782211.0
  yarn add @salutejs/plasma-tokens-b2c@0.18.1-canary.343.4182782211.0
  yarn add @salutejs/plasma-tokens-web@1.24.1-canary.343.4182782211.0
  yarn add @salutejs/plasma-tokens@1.35.1-canary.343.4182782211.0
  yarn add @salutejs/plasma-ui@1.168.2-canary.343.4182782211.0
  yarn add @salutejs/plasma-web@1.166.2-canary.343.4182782211.0
  yarn add @salutejs/plasma-cy-utils@0.42.1-canary.343.4182782211.0
  yarn add @salutejs/plasma-sb-utils@0.94.1-canary.343.4182782211.0
  yarn add @salutejs/plasma-tokens-android@2.42.1-canary.343.4182782211.0
  yarn add @salutejs/plasma-tokens-ios-swift@2.42.1-canary.343.4182782211.0
  yarn add @salutejs/plasma-tokens-utils@0.17.1-canary.343.4182782211.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-343/

File: website/plasma-ui-docs/docs/design/tokens.mdx [hunk 1]
```diff
@@ -33,6 +35,36 @@ export default function App() {
...
+```jsx title="App.tsx"
+import React from 'react';
+import styled from 'styled-components';
+
+import { bodyM } from '@salutejs/plasma-typo';
+import { darkSbermarket } from '@salutejs/plasma-tokens';
+import { surfaceAccent, textPrimary } from '@salutejs/plasma-tokens/brands/sbermarket';
+
+const AppStyled = styled.div`
+    ${darkSbermarket[":root"]}
```
Comment (Yeti-or) [inline website/plasma-ui-docs/docs/design/tokens.mdx:51]: Вот это наши костыли же

**Review by Yeti-or — APPROVED** [review]
Мне тут что то не нравится но объяснить что не могу :/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-343/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-343/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-343/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-343/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-343/temple-storybook/

**Review by Yeti-or — APPROVED** [review]

File: website/plasma-ui-docs/docs/design/tokens.mdx (hunk 1 above)
Comment (neretin-trike) [reply website/plasma-ui-docs/docs/design/tokens.mdx:51]: в итоге договорились, что разделю пример

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-343/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-343/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-343/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-343/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-343/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-343/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-343/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Review by Yakutoc — APPROVED** [review]

File: packages/plasma-tokens/package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "types": "index.d.ts",
     "scripts": {
         "prepare": "npm run build",
-        "prebuild": "rm -rf ./build ./colors ./es ./properties ./src ./themes ./themesValues ./typo ./typography ./typographyValues ./index.* ./theme.json",
```
Comment (Yeti-or) [inline packages/plasma-tokens/package.json:11]: А в files не надо добавить ?

**Review by Yeti-or — CHANGES_REQUESTED** [review]

File: packages/plasma-tokens/package.json (hunk 2 above)
Comment (neretin-trike) [reply packages/plasma-tokens/package.json:11]: дак я files же удалил, вместо этого у нас сейчас .npmignore

Comment (neretin-trike) [reply packages/plasma-tokens/package.json:11]: надо :/, спасибо

File: website/plasma-ui-docs/docs/design/tokens.mdx [hunk 3]
```diff
@@ -33,6 +35,40 @@ export default function App() {
...
+
+```jsx title="App.tsx"
+import React from 'react';
+import styled, { createGlobalStyle } from 'styled-components';
+
+import { bodyM } from '@salutejs/plasma-typo';
+import { darkSbermarket } from '@salutejs/plasma-tokens';
+import { surfaceAccent, textPrimary } from '@salutejs/plasma-tokens/brands/sbermarket';
+
+const Theme = createGlobalStyle(darkSbermarket);
```
Comment (Yeti-or) [inline website/plasma-ui-docs/docs/design/tokens.mdx:50]: 👍

**Review by Yeti-or — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-343/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-343/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-343/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-343/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-343/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-343/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-343/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Pull request merged**